plaso/cli/tools.py

```diff
@@ -223,8 +230,9 @@ def AddLogFileOptions(self, argument_group):
     argument_group.add_argument(
         '--logfile', '--log_file', '--log-file', action='store',
         metavar='FILENAME', dest='log_file', type=str, default='', help=(
-            'If defined all log messages will be redirected to this file '
-            'instead the default STDERR.'))
+            'Path of the file in which to store log messages, by default '
+            'this file will be named: {0:s}-YYYYMMDDThhmmss.log').format(
```

Comment:
Per line 152, it's actually .log.gz. Please harmonize the values here.

I think it's also worth mentioning that if the file name ends in .gz, the file will be compressed, but otherwise it'll be uncompressed.

Reply:
ack, changes made